stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -186,6 +186,16 @@ IndexStyle(::Type{<:AdjOrTransAbsMat}) = IndexCartesian()
 convert(::Type{Adjoint{T,S}}, A::Adjoint) where {T,S} = Adjoint{T,S}(convert(S, A.parent))
 convert(::Type{Transpose{T,S}}, A::Transpose) where {T,S} = Transpose{T,S}(convert(S, A.parent))
 
+# Strides and pointer for transposed strided arrays — but only if the elements are actually stored in memory
+Base.strides(A::Adjoint{<:Real, <:StridedVector}) = (stride(A.parent, 2), stride(A.parent, 1))
+Base.strides(A::Transpose{<:Any, <:StridedVector}) = (stride(A.parent, 2), stride(A.parent, 1))
+# For matrices it's slightly faster to use reverse and avoid calling stride twice
+Base.strides(A::Adjoint{<:Real, <:StridedMatrix}) = reverse(strides(A.parent))
+Base.strides(A::Transpose{<:Any, <:StridedMatrix}) = reverse(strides(A.parent))
+
+Base.pointer(A::Adjoint{<:Real, <:Union{StridedVector,StridedMatrix}) = pointer(A.parent)
```

Comment:
Isn't it better to overload `unsafe_convert`?

Reply:
Yes, I suppose so, thanks!